pkg/api/v1beta3/conversion.go

```diff
@@ -39,6 +39,19 @@ func init() {
 		// If one of the conversion functions is malformed, detect it immediately.
 		panic(err)
 	}
+	err = newer.Scheme.AddFieldLabelConversionFunc("v1beta3", "Minion",
+		func(label, value string) (string, string, error) {
+			switch label {
+			case "name":
```

Comment:
Reviewers: just name by itself is incorrect. The field labels should match the access you would do to get the field in the object. "name" in v1beta3 is stored in .metadata.name. This is an api error.